src/uwtools/tests/test_cli.py

```diff
@@ -276,16 +276,19 @@ def test__dict_from_key_eq_val_strings():
 
 
 def test__dispatch_chgres_cube():
+    cycle = (dt.datetime.now(),)
```

Comment:
I'm confused by this tuple and the next one.

Reply:
I think there was a rogue comma, which made the value technically a tuple, which `black` then dutifully wrapped in parens to make that explicit. I'll clean it up. Thanks.